tnp/templates/consent/change_password.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'base.html' %}
+
+{% block title %} Change Password {% endblock %}
+
+{% block custom_css %}
+	<link type="text/css" rel="stylesheet" href="/static/css/datepicker.css" media="screen,projection" />
+	<link type="text/css" rel="stylesheet" href="/static/css/form.css" media="screen,projection" />
+{% endblock %}
+
+
+{% block body_block %} 
+	<div id="content" style="min-height: calc(100vh - 120px);">
+		<div class="container">
+			<div class="section" id="form_errors"> 
+				{% if messages %}
+  				<ul class="collection red-text">
+    				{% for message in messages %}
+							<li class="collection-item">{{ message }} </li>
+    					{% endfor %}
+  							</ul>
+								{% endif %}
+			</div>
+
+				<div class="row">
+					<form method="post" id="create_account_form" enctype="multipart/form-data">
+						{% csrf_token %}
+						<div class="card grey lighten-5 grey-text text-darken-2 z-depth-2 hoverable">
+	                    	<div class="card-content grey darken-2 grey-text text-lighten-5 z-depth-2">
+	                      		<span class="card-title"> Preliminary Details </span>
+	                    	</div>
+
+	                    	<div class="section"> </div>
+							  {{ form }}
+						</div>
+
+						<center>
+		                    <button class="waves-effect waves-light btn-large grey darken-2 grey-text text-lighten-5 hoverable" id="create_account_button" type="submit"> Change Password </button>
+		                </center>
+					</form>
+				</div>
+			
+		</div>
+	</div>
+{% endblock %}
+
+
+
+
+
+
+{% block custom_js %}
+	<script type="text/javascript" src="/static/js/consent/create_account.js"></script>
+{% endblock %}
```

Comment:
I have developed this as a test .UI thing is remaining .I will try to do it quickly